packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -435,6 +438,40 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       erc20Abi: ERC20.abi,
       blockFinder: getSharedBlockFinder(web3)
     });
+  } else if (config.type === "frm") {
+    const requiredFields = ["perpetualAddress"];
+    if (isMissingField(config, requiredFields, logger)) {
+      return null;
+    }
+
+    logger.debug({
+      at: "createPriceFeed",
+      message: "Creating FundingRateMultiplierPriceFeed",
+      config
+    });
+
+    // Grab multicall address from network (will be undefined if any lookups fail).
+    const networkId = await web3.eth.net.getId();
```

Comment:
Can you skip this part if `config.multicallAddress` is provided by the caller?

Reply:
Yep, I'll change this.

Reply:
Done.